README.md

```diff
@@ -1,6 +1,6 @@
 # Babble
```

Comment:
If possible we could replace this header with the banner in the Figma. I can send it if you don't have it! 

Typically I'll insert it using HTML... :sunglasses: or it can be done in markdown too! :smile: 

```
<p align="center">
  <img src="./images/Banner.png" alt="Banner" />
<p>
---

or

[Banner](./images/Banner.png)
```
Doing this might retain the visual appeal of having the landing page image at the top, without repeating it twice.

Reply:
Great idea with the banner! I think we should add that

Reply:
Yeah the banner would a great addition!